setup.py

```diff
@@ -46,14 +47,10 @@
     "pytest-cov",
     # TODO: update config so coveralls 3 works
     "coveralls<3",
+    "altair",
```

Comment:
please add a comment explainign that altair is needed for interactive confusion matrix